recipes/pew/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pew" %}
+{% set version = "1.1.5" %}
+{% set sha256 = "75dbc3d91561988b0f19bf8d3b3ba777fff5c91747ca02034094a4887be2edf1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/berdario/pew/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - pew = pew.pew:pew
+
+requirements:
+  build:
+    - python
+    - pip
+
+  run:
+    - python
+    - virtualenv >=1.11
+    - virtualenv-clone >=0.2.5
+    - setuptools >=17.1
```

Comment:
I think setuptools should be in ``build`` or are you using this library in your code besides the ``setup.py`` ?